### DISCUSSION TITLE
How to open Reveal.js without navbar

### DISCUSSION THREAD

**Discussion by NicHaley** [hakimel/reveal.js#3406] [Q&A]
I came across Reveal.js watching a tutorial on https://threejs-journey.com/

In the tutorial, the Reveal.js window is just a window -- there is no browser navbar etc. For presentations this would be really nice for reducing visual noise. how do I achieve this?

<img width="651" alt="Screenshot 2023-05-10 at 12 37 47 PM" src="https://github.com/hakimel/reveal.js/assets/9045634/42bade83-ad92-4d48-8877-7b0d0f75cfff">

**Comment by hakimel** [comment]
That looks like a window opened with `window.open`. That means it's only possible for popup windows—not the primary browser window. Also I think Safari is the only browser that hides the address bar in popups.
